include/netutils/webclient.h

```diff
@@ -191,13 +191,17 @@ struct webclient_context
    *   url          - A pointer to a string containing the full URL.
    *                  (e.g., http://www.nutt.org/index.html, or
    *                   http://192.168.23.1:80/index.html)
+   *   unix_socket  - If not NULL, the path to an AF_LOCAL socket.
    *   headers      - An array of pointers to the extra headers.
    *   nheaders     - The number of elements in the "headers" array.
    *   bodylen      - The size of the request body.
    */
 
   FAR const char *method;
   FAR const char *url;
+#if defined(CONFIG_WEBCLIENT_NET_LOCAL)
+  FAR const char *unix_socket;
```

Comment:
change to unpath? socket normally reserve to handle.

Reply:
i named it after the curl option.
```
--unix-socket PATH    Connect through this Unix domain socket
```

how about `unix_socket_path` then?
`unpath` sounds a bit cryptic to me.

Reply:
Ok.